.github/actions/workflow-vars/action.yml

```diff
@@ -5,15 +5,23 @@ inputs:
     description: Don't use changing vars like SHA commit
     # TODO: Change that to real bool when it's supported in GHA.
     default: "false"
+    required: true
   setup-flow:
     description: The chart setup flow either "install" or "upgrade".
     default: "install"
+    required: true
```

Comment:
if you have a default, it's not gonna be required is it?

Also fun fact, seems like required only takes effect in the case of `workflow_dispatch` --> https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-specifying-inputs

> Note: Workflows using required: true will not automatically return an error if the input is not specified for events that automatically trigger workflow runs. If you set required: true in your workflow file and are using workflow_dispatch to manually run the workflow, you will be required to specify inputs on GitHub.